### PR TITLE
Start working on the metrics collection docs.

### PR DESCRIPTION
**Pull request by dualspiral**
After the merge of the Metrics Collection API for Ore, it's prudent that we document it!

I've added the basics so far, but I have more to do. Specifically:

* Add docs on what this section is in a bit more detail for a server admin, so they know that plugins downloaded from Ore have metrics collection off by default (maybe? Wanted? Where should it go?)
* Add docs on the API for plugin developers, including a link to the Ore guidelines (and maybe @mbax will want to chime in with thoughts on what we should add)

I'll do this over the next few days, uploading what I have now. Mostly want this visible so you know it's being worked on.

(Some line endings seem to have changed, probably because I'm on Windows at the moment. I'll have to look into that...)

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/bcdfe7e/.

Here are some links to the pages that were modified:

- added: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/bcdfe7e/plugin/metrics.html
- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/bcdfe7e/plugin/index.html
- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/bcdfe7e/server/getting-started/configuration/sponge-conf.html
- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/bcdfe7e/server/spongineer/commands.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/755#issuecomment-416296264), not to the direct url to the preview._

**Comment by mbax** [comment]
May be worth documenting the arguments, since they're not accessible from console.

**Review by ST-DDT — APPROVED** [review]

**Comment by ST-DDT** [comment]
If we add the params for that command we have to add it for all. What are you referring to if you say that the arguments are not accessible from the console?
I just tried it with auto completion and it worked perfectly fine. The only thing that was not possible was setting the default.

**Comment by ST-DDT** [comment]
* [x] Please resolve merge conflicts.

**Comment by dualspiral** [comment]
I will when I get around to finishing it, there's more I want to add to the docs about it.

**Comment by ST-DDT** [comment]
Maybe we should add a sentence that server owners shouldn't be bothered or forced to enable metrics. But I guess we could also leave this up to the server owners to uninstall the offending mod. However, that would leave a bad aftertaste for the users of our platform.

**Comment by dualspiral** [comment]
They certainly shouldn’t be forced to, if they are forced, that’s against the guidelines and so wouldn’t get through anyway. As for bothering... I would suspect the community will be self policing in that matter and vote with their feet. I don’t think they should, but if that’s the policy then we should put that in the guidelines - the documents here are techincal.

I don’t know if we should have a separate page on the docs for metrics and how to enable/disable for server owners. If not, then this is pretty much there. If so, I can write one.

File: source/plugin/metrics.rst
Comment (Inscrutable) [inline source/plugin/metrics.rst]: playes -> players

File: source/plugin/metrics.rst
Comment (Inscrutable) [inline source/plugin/metrics.rst]: I think there is an "only" missing, as in "... may *only* collect metrics if ..."

**Review by Inscrutable — COMMENTED** [review]
Indeed. Will you now remove the [WIP] ?

**Comment by dualspiral** [comment]
If we want a server owner facing page, I can write one in a future PR.

Note that some of the Javadocs links will not work until API 7.1 is released and the JDs are deployed. You might wish to hold off merging this until then, hopefully this won't be too far away.

**Comment by dualspiral** [comment]
Now API 7.1 has been released, this should be good to go. I've made a small addition, missed out an import in the code block on the Metrics page.

**Review by ST-DDT — APPROVED** [review]

**Review by mbax — APPROVED** [review]
:+1:

**Pull request merged**
